driver/nbd_protocol.c

```diff
@@ -228,14 +230,14 @@ NbdNegotiate(_In_ INT* Pfd,
     RtlZeroMemory(Buf, 8);
     NbdReadExact(Fd, Buf, 8, &status);
     if (strcmp(Buf, INIT_PASSWD)) {
-        WNBD_LOG_ERROR("INIT_PASSWD");
+        WNBD_LOG_INFO("INIT_PASSWD");
```

Comment:
Let's also update the message, maybe something like "Received NBD INIT_PASSWD". Also, I guess it can be a debug message.